test/FsAutoComplete.Tests.Lsp/InlayHintTests.fs

```diff
@@ -108,12 +157,321 @@ let tests state =
+          let range = 2
+
+          $0f range$0
+          """
+          []
+      testCaseAsync "hide: arg is postfix of param with trailing '" <|
+        InlayHints.checkRange server
+          """
+          let f exactRange' = ()
+          let range = 2
+
+          $0f range$0
+          """
+          []
+      testCaseAsync "hide: arg is prefix of param with trailing ' in arg" <|
+        InlayHints.checkRange server
+          """
+          let f rangeCoveringExpr = ()
+          let range' = 2
+
+          $0f range'$0
+          """
+          []
+
+      testCaseAsync "hide: param prefix of arg" <|
+        InlayHints.checkRange server
+          """
+          let f range = ()
+          let rangeCoveringExpr = 2
+
+          $0f rangeCoveringExpr$0
+          """
+          []
+      testCaseAsync "hide: param postfix of arg" <|
+        InlayHints.checkRange server
+          """
+          let f range = ()
+          let exactRange = 2
+
+          $0f exactRange$0
+          """
+          []
+
+      testCaseAsync "hide: arg is field access with same name as param (upper case start)" <|
+        InlayHints.checkRange server
+          """
+          type Data = {
+            Range: int
+          }
+          let f range = ()
+          let data: Data = { Range = 2 }
+
+          $0f data.Range$0
+          """
+          []
+      testCaseAsync "hide: arg is field access with same name as param (lower case start)" <|
+        InlayHints.checkRange server
+          """
+          type Data = {
+            range: int
+          }
+          let f range = ()
+          let data: Data = { range = 2 }
+
+          $0f data.range$0
+          """
+          []
+      testCaseAsync "hide: arg is field access prefix of param (upper case start)" <|
+        InlayHints.checkRange server
+          """
+          type Data = {
+            Range: int
+          }
+          let f rangeCoveringExpr = ()
+          let data: Data = { Range = 2 }
+
+          $0f data.Range$0
+          """
+          []
+      testCaseAsync "hide: arg is field access, param is prefix of arg" <|
+        InlayHints.checkRange server
+          """
+          type Data = {
+            RangeCoveringExpr: int
+          }
+          let f range = ()
+          let data: Data = { RangeCoveringExpr = 2 }
+
+          $0f data.RangeCoveringExpr$0
+          """
+          []
+
+      testCaseAsync "hide: arg in parens same as param" <|
+        InlayHints.checkRange server
+          """
+          let f alpha = ()
+          let alpha = 42
+
+          $0f (alpha)$0
+          """
+          [  ]
+      testCaseAsync "hide: arg in parens and spaces same as param" <|
+        InlayHints.checkRange server
+          """
+          let f alpha = ()
+          let alpha = 42
+
+          $0f ( alpha )$0
+          """
+          [  ]
+      //todo: or hide? based on: what is last? but then (`alpha <| 1`, `1 |> alpha 2`, etc?) -> too complex to detect
```

Comment:
I agree with passing on these for now and just letting the parameter names show.